src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  width: $card-size;
+  border-radius: 5px;
+  border: 1px #F3F3F3 solid;
+
+  transition: transform 300ms ease;
+
+  &__container {
+    margin-right: 16px;
+    margin-left: 16px;
+  }
+
+  &__product-photo {
+    display: flex;
+    justify-content: center;
+  }
+
+	&__buy-batton {
+    height: 40px;
+    width: 100%;
+    padding: 0;
+
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #fff;
+    background: $link-active-color;
+    border-radius: 5px;
+    border-style: none;
+    margin-bottom: 16px;
+	}
```

Comment:
```suggestion
  &__buy-button {
    height: 40px;
    width: 100%;
    padding: 0;

    font-size: 14px;
    line-height: 16px;
    text-transform: uppercase;
    color: #fff;
    background: $link-active-color;
    border-radius: 5px;
    border-style: none;
    margin-bottom: 16px;
  }
```
format your CSS code - use correct indentations
also, typo in `button`

Reply:
Everything looks fine on my computer, this may be  GitHub formatting does it

<img width="426" alt="Знімок екрана 2024-02-15 о 19 35 48" src="https://github.com/mate-academy/layout_catalog/assets/143593584/e96f2b2c-a241-49df-a313-f28141ba26c7">